sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -325,7 +325,9 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
     );
 
     try {
-      await this._init(sendRequestOptions?.abortSignal);
+      if (!this.isOpen()) {
+        await this._init(sendRequestOptions?.abortSignal);
+      }
```

Comment:
This fix should work. 

I think the issue here is that the two event listeners that are being added should have been passed in as part of the respective options for their links, much like this one is for session error:

https://github.com/Azure/azure-sdk-for-js/blob/8fb7f4e92e7774c6c08ada92198451900d990718/sdk/servicebus/service-bus/src/core/managementClient.ts#L232

(and yes, I totally missed this during my refactor. We can add in some testing for this and possibly a longer term fix as a later PR).

Reply:
@richardpark-msft, Can you log an issue for the said longer term fix or do we want to re-use the existing one?

Reply:
> I think the issue here is that the two event listeners that are being added should have been passed in as part of the respective options for their links, much like this one is for session error

Agree with that. I did think in those lines, though didn't spend much time on how. Moreover, initializing the link for each request shouldn't happen even if we tie the listeners and since that fixes the current issue, went ahead with it.